test/reducers/measures_test.js

```diff
@@ -0,0 +1,23 @@
+import { expect } from '../test_helper';
+
+import { flattenCategories, flattenMeasures } from '../../src/reducers/measures';
+
+describe('Measure reducers', () => {
+  it('should flatten categories', () => {
```

Comment:
I have tracked down the issue. The problem is that I'm using the stable version of node (4.5.0), but node doesn't support `includes` until version 6 (current is 6.5.0). This is the function in question in `/src/reducers/measures.js`:

```
export function flattenCategories(measures) {
  let categories = [];
  measures.forEach((m) => {
    if (! categories.includes(m.category)) {
      categories.push(m.category);
    }
  });
  return categories.sort();
}
```

While anyone using the current node version would be fine with this, I'd suggest changing `! categories.includes(m.category)` to `categories.indexOf(m.category) === -1` in order to handle users (like me) who are using the older stable node version.